app/src/main/kotlin/com/simplemobiletools/flashlight/activities/SettingsActivity.kt

```diff
@@ -2,145 +2,64 @@ package com.simplemobiletools.flashlight.activities
 
 import android.content.Intent
 import android.os.Bundle
-import com.simplemobiletools.commons.extensions.*
+import androidx.activity.compose.setContent
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.simplemobiletools.commons.compose.extensions.enableEdgeToEdgeSimple
+import com.simplemobiletools.commons.compose.theme.AppThemeSurface
 import com.simplemobiletools.commons.helpers.IS_CUSTOMIZING_COLORS
-import com.simplemobiletools.commons.helpers.NavigationIcon
-import com.simplemobiletools.commons.helpers.isTiramisuPlus
-import com.simplemobiletools.flashlight.databinding.ActivitySettingsBinding
 import com.simplemobiletools.flashlight.extensions.config
+import com.simplemobiletools.flashlight.screens.SettingsScreen
 import java.util.Locale
-import kotlin.system.exitProcess
 
 class SettingsActivity : SimpleActivity() {
```

Comment:
Make sure that every activity you're using is inheriting from `ComponentActivity` keep in mind that some of the functionalities you will have to extract as extension functions.

Reply:
Some of these functionalities apply to all apps (like starting about activity and similar things which were in `BaseSimpleActivity`). Should we move these extensions to Commons, to prevent each app redefining them?